graphql/resolvers/operations.go

```diff
@@ -6,39 +6,74 @@ import (
 	"time"
 
 	"github.com/MichaelMure/git-bug/bug"
+	"github.com/MichaelMure/git-bug/graphql/graph"
 	"github.com/MichaelMure/git-bug/graphql/models"
 )
 
+var _ graph.CreateOperationResolver = createOperationResolver{}
+
 type createOperationResolver struct{}
 
+func (createOperationResolver) ID(ctx context.Context, obj *bug.CreateOperation) (string, error) {
+	return obj.Id().String(), nil
+}
+
 func (createOperationResolver) Date(ctx context.Context, obj *bug.CreateOperation) (*time.Time, error) {
 	t := obj.Time()
 	return &t, nil
 }
 
+var _ graph.AddCommentOperationResolver = addCommentOperationResolver{}
+
 type addCommentOperationResolver struct{}
 
+func (addCommentOperationResolver) ID(ctx context.Context, obj *bug.AddCommentOperation) (string, error) {
+	return obj.Id().String(), nil
+}
+
 func (addCommentOperationResolver) Date(ctx context.Context, obj *bug.AddCommentOperation) (*time.Time, error) {
 	t := obj.Time()
 	return &t, nil
 }
 
+var _ graph.EditCommentOperationResolver = editCommentOperationResolver{}
+
 type editCommentOperationResolver struct{}
 
+func (editCommentOperationResolver) ID(ctx context.Context, obj *bug.EditCommentOperation) (string, error) {
+	return obj.Id().String(), nil
+}
+
+func (editCommentOperationResolver) Target(ctx context.Context, obj *bug.EditCommentOperation) (string, error) {
+	panic("implement me")
```

Comment:
@A-Hilaly  :-O

Reply:
Who did this x) ?? Is it complex to implement ?